scripts/setup-centos8.sh

```diff
@@ -50,8 +50,9 @@ function install_build_prerequisites {
   dnf_install epel-release dnf-plugins-core # For ccache, ninja
   dnf config-manager --set-enabled powertools
   dnf update -y
-  dnf_install ninja-build cmake curl ccache gcc-toolset-9 git wget which
-  dnf_install autoconf automake python39 python39-devel python39-pip libtool
+  dnf_install ninja-build curl ccache gcc-toolset-9 git wget which
+  dnf_install autoconf automake python39 python39-devel python39-pip libtool c-ares-devel
+  pip3.9 install cmake==3.28.3
 }
```

Comment:
the c-ares should also be installed in adapters.sh
For the pip command it should be fine without the version but being explicit doesn't hurt.

Reply:
Yeah, I prefer to be explicit, future versions of cmake may be added to pip and may introduce breaking changes.